[2024-11-25T20:52:56.490+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: kafka_to_snowflake_pipeline.upload_transformed_data_to_stage manual__2024-11-25T20:51:42.977665+00:00 [queued]>
[2024-11-25T20:52:56.500+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: kafka_to_snowflake_pipeline.upload_transformed_data_to_stage manual__2024-11-25T20:51:42.977665+00:00 [queued]>
[2024-11-25T20:52:56.500+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2024-11-25T20:52:56.501+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 2
[2024-11-25T20:52:56.501+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2024-11-25T20:52:56.512+0000] {taskinstance.py:1304} INFO - Executing <Task(SnowflakeOperator): upload_transformed_data_to_stage> on 2024-11-25 20:51:42.977665+00:00
[2024-11-25T20:52:56.551+0000] {standard_task_runner.py:55} INFO - Started process 1574 to run task
[2024-11-25T20:52:56.572+0000] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'kafka_to_snowflake_pipeline', 'upload_transformed_data_to_stage', 'manual__2024-11-25T20:51:42.977665+00:00', '--job-id', '62', '--raw', '--subdir', 'DAGS_FOLDER/kafka_to_snowflake_etl_dag.py', '--cfg-path', '/tmp/tmpvtkkn_vb']
[2024-11-25T20:52:56.582+0000] {standard_task_runner.py:83} INFO - Job 62: Subtask upload_transformed_data_to_stage
[2024-11-25T20:52:56.768+0000] {task_command.py:389} INFO - Running <TaskInstance: kafka_to_snowflake_pipeline.upload_transformed_data_to_stage manual__2024-11-25T20:51:42.977665+00:00 [running]> on host f25f59e56f04
[2024-11-25T20:52:56.874+0000] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Vaibhav Bansal
AIRFLOW_CTX_DAG_ID=kafka_to_snowflake_pipeline
AIRFLOW_CTX_TASK_ID=upload_transformed_data_to_stage
AIRFLOW_CTX_EXECUTION_DATE=2024-11-25T20:51:42.977665+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-11-25T20:51:42.977665+00:00
[2024-11-25T20:52:56.875+0000] {sql.py:253} INFO - Executing:  
        PUT file:///opt/airflow/shared/transformed_data.parquet/part-00000-56082414-f17f-4418-85af-5103d26d4699-c000.snappy.parquet @~/stage/ AUTO_COMPRESS=FALSE;
        
[2024-11-25T20:52:56.886+0000] {base.py:73} INFO - Using connection ID 'snowflake_default' for task execution.
[2024-11-25T20:52:57.133+0000] {base.py:73} INFO - Using connection ID 'snowflake_default' for task execution.
[2024-11-25T20:52:57.144+0000] {connection.py:414} INFO - Snowflake Connector for Python Version: 3.12.3, Python Version: 3.8.15, Platform: Linux-6.6.22-linuxkit-aarch64-with-glibc2.17
[2024-11-25T20:52:57.146+0000] {connection.py:1197} INFO - Connecting to GLOBAL Snowflake domain
[2024-11-25T20:52:57.146+0000] {connection.py:1278} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-25T20:52:57.146+0000] {connection.py:1288} INFO - THIS CONNECTION IS IN INSECURE MODE. IT MEANS THE CERTIFICATE WILL BE VALIDATED BUT THE CERTIFICATE REVOCATION STATUS WILL NOT BE CHECKED.
[2024-11-25T20:52:57.322+0000] {ssl_wrap_socket.py:101} INFO - THIS CONNECTION IS IN INSECURE MODE. IT MEANS THE CERTIFICATE WILL BE VALIDATED BUT THE CERTIFICATE REVOCATION STATUS WILL NOT BE CHECKED.
[2024-11-25T20:52:57.624+0000] {cursor.py:1166} INFO - Number of results in first chunk: 1
[2024-11-25T20:52:57.624+0000] {sql.py:364} INFO - Running statement: PUT file:///opt/airflow/shared/transformed_data.parquet/part-00000-56082414-f17f-4418-85af-5103d26d4699-c000.snappy.parquet @~/stage/ AUTO_COMPRESS=FALSE, parameters: None
[2024-11-25T20:52:57.866+0000] {ssl_wrap_socket.py:101} INFO - THIS CONNECTION IS IN INSECURE MODE. IT MEANS THE CERTIFICATE WILL BE VALIDATED BUT THE CERTIFICATE REVOCATION STATUS WILL NOT BE CHECKED.
[2024-11-25T20:52:58.048+0000] {sql.py:373} INFO - Rows affected: 1
[2024-11-25T20:52:58.049+0000] {snowflake.py:400} INFO - Rows affected: 1
[2024-11-25T20:52:58.049+0000] {snowflake.py:401} INFO - Snowflake query id: 01b89d64-0002-35a5-0005-d39200032652
[2024-11-25T20:52:58.220+0000] {cursor.py:1166} INFO - Number of results in first chunk: 1
[2024-11-25T20:52:58.221+0000] {connection.py:789} INFO - closed
[2024-11-25T20:52:58.281+0000] {connection.py:795} INFO - No async queries seem to be running, deleting session
[2024-11-25T20:52:58.389+0000] {taskinstance.py:1322} INFO - Marking task as SUCCESS. dag_id=kafka_to_snowflake_pipeline, task_id=upload_transformed_data_to_stage, execution_date=20241125T205142, start_date=20241125T205256, end_date=20241125T205258
[2024-11-25T20:52:58.469+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2024-11-25T20:52:58.538+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
